js/apps/admin-ui/src/realm-settings/user-profile/attribute/AddTranslationsDialog.tsx

```diff
@@ -0,0 +1,363 @@
+                                {locale === defaultLocales.toString() && (
+                                  <>
+                                    <Label className="pf-u-ml-xs" color="blue">
+                                      {t("defaultLanguage")}
+                                    </Label>
+                                    <Label
+                                      color="red"
+                                      icon={<InfoCircleIcon />}
+                                    >
+                                      {t("requiredLanguage")}
+                                    </Label>
+                                  </>
+                                )}
+                              </FormGroup>
+                            </Td>
+                            <Td>
+                              {locale === defaultLocales.toString() && (
+                                <FormGroup
+                                  fieldId="kc-translationValue"
+                                  helperText={
+                                    locale === defaultLocales.toString() &&
+                                    t("addTranslationDialogHelperText")
+                                  }
+                                >
+                                  <Controller
+                                    name={`translations.${rowIndex}`}
+                                    control={control}
+                                    rules={{ required: true }}
+                                    render={({ field }) => (
+                                      <KeycloakTextInput
+                                        id="translationValue"
+                                        {...field.value}
+                                        aria-label={t("translationValue")}
+                                        data-testid="translation-value"
+                                        validated={
+                                          isError
+                                            ? ValidatedOptions.error
+                                            : "default"
+                                        }
+                                        onChange={(e) => {
+                                          const updatedTranslation = {
+                                            locale,
+                                            value: (
+                                              e.target as HTMLInputElement
+                                            ).value,
+                                          };
+                                          setValue(
+                                            `translations.${rowIndex}`,
+                                            updatedTranslation,
+                                          );
+                                          setIsError(
+                                            updatedTranslation.value === "",
+                                          );
```

Comment:
why not make the field required?

Reply:
This implementation is based on the design. Originally I was supposed to use warning, but we decided to use error instead.

![Screenshot 2024-02-28 at 15 15 00](https://github.com/keycloak/keycloak/assets/4890675/c43ab3ad-e656-4b14-821e-15f3cb282c87)